### PR TITLE
Improve Video Grid UI

### PR DESCRIPTION
**Pull request by clloyd**
screen shot 2016-12-05 at 11 59 40

🐧  🐧  🐧  🐧

### CODE REVIEW

**Comment by clloyd** [comment]
@akash1810

File: public/video-ui/src/components/Videos/VideoItem.js [hunk 1]
```diff
@@ -3,19 +3,41 @@ import {Link} from 'react-router';
 
 export default class VideoItem extends React.Component {
 
+  renderActiveAssetName() {
+
+    if (this.props.video.activeVersion && this.props.video.assets.length) {
+      const activeAssets = this.props.video.assets.filter((asset) => asset.version === this.props.video.activeVersion)
+      if (activeAssets.length !== -1) {
```
Comment (akash1810) [inline public/video-ui/src/components/Videos/VideoItem.js]: does this work? wouldn't `.filter` return an empty array if nothing can be found?

File: public/video-ui/styles/layout/_grid.scss [hunk 2]
```diff
@@ -11,38 +10,96 @@
...
+  text-align: center;
 }
 
 .grid__item {
-  flex-basis: 25%;
-  padding: 5px;
+  display: inline-block;
+
+  position: relative;
+  border: 3px solid #3e3e3e;
```
Comment (akash1810) [inline public/video-ui/styles/layout/_grid.scss]: can we use a variable here?

File: public/video-ui/src/components/Videos/VideoItem.js (hunk 1 above)
Comment (clloyd) [inline public/video-ui/src/components/Videos/VideoItem.js]: Whoops wrong syntax should be `!activeAssets.length`

**Comment by clloyd** [comment]
@akash1810 fixed your comments, but accidentally pushed a fix for select boxes showing id's rather than titles when in display mode

**Pull request merged**